locale/en/ManageIQ_UI_Classic.po

```diff
@@ -1904,19 +1856,16 @@ msgstr ""
 msgid "Add Interface to Router \"%{name}\""
 msgstr ""
 
-msgid "Add Interface to selected Router"
-msgstr ""
-
 msgid "Add Interface to this Router"
 msgstr ""
 
-msgid "Add JDBC Driver"
+msgid "Add Method"
 msgstr ""
 
-msgid "Add Middleware Deployment"
+msgid "Add Network"
 msgstr ""
 
-msgid "Add New %{model}"
```

Comment:
This msgid disappeared, causing the test failure.

But `Add New %{table}` still exists, so...

```diff
diff --git a/spec/helpers/application_helper/toolbar_builder_spec.rb b/spec/helpers/application_helper/toolbar_builder_spec.rb
index 78ace0a33..6bd7b05b4 100644
--- a/spec/helpers/application_helper/toolbar_builder_spec.rb
+++ b/spec/helpers/application_helper/toolbar_builder_spec.rb
@@ -271,7 +271,7 @@ describe ApplicationHelper, "::ToolbarBuilder" do
       it "does delayed translation of text title and confirm strings" do
         %i(text title confirm).each do |key|
           @input[key] = proc do
-            _("Add New %{model}") % {:model => 'Model'}
+            _("Add New %{table}") % {:table => 'Model'}
           end
         end
         FastGettext.locale = 'ja'
```

should be enough for that spec.

Reply:
Yep, fixed in https://github.com/ManageIQ/manageiq-ui-classic/pull/4740